src/worker.c

```diff
@@ -430,16 +423,14 @@ _PG_init(void)
                       "configuration variable in postgresql.conf."));
   }
 
-  MemSet(&worker, 0, sizeof(BackgroundWorker));
-  worker.bgw_flags = BGWORKER_SHMEM_ACCESS | BGWORKER_BACKEND_DATABASE_CONNECTION;
-  worker.bgw_start_time = BgWorkerStart_RecoveryFinished;
-  snprintf(worker.bgw_library_name, BGW_MAXLEN, "pg_net");
-  snprintf(worker.bgw_function_name, BGW_MAXLEN, "pg_net_worker");
-  snprintf(worker.bgw_name, BGW_MAXLEN, "pg_net " EXTVERSION " worker");
-  worker.bgw_restart_time = 1;
-  worker.bgw_main_arg = (Datum) 0;
-  worker.bgw_notify_pid = 0;
-  RegisterBackgroundWorker(&worker);
+  RegisterBackgroundWorker(&(BackgroundWorker){
```

Comment:
IMO previous state of pg_init() was completely ok and also was easier to read.

Reply:
The new way using C99 designated initializer requires fewer LOC and avoids the needed to `memset` to 0. Readability is highly subjective but fewer LOCs in this case is a win IMO.